extensions/qcom/GLSL_QCOM_tile_shading.txt

```diff
@@ -0,0 +1,326 @@
+      tile_attachmentQCOM
+
+
+    Add a subsection to Section 4.4, Layout Qualifiers (p. 97)
+
+    4.4.X Tile Shading Layout Qualifiers
+
+    Tile image attachments are only available when targeting Vulkan.
+
+    The layout qualifiers for declaration of tile image attachment variables are:
+
+    layout-qualifier-id:
+        set      = layout-qualifier-value
+        binding  = layout-qualifier-value
+        tile_attachmentQCOM
+
+    Tile image attachment variables are declared with the 2D image/texture/sampler types.
+    They must be declared with the set and binding layout qualifiers and with the
+    tile_attachmentQCOM layout qualifier.  Otherwise, a compile-time error results.
+
+    For example:
+
+     layout( set = 0, binding = 0, tile_attachmentQCOM, rgba32f ) uniform highp image2D color0;
+
+    See the Vulkan API documentation for details on mapping from tile image
+    attachment variables to color/depth/stencil/input attachments.
+
+
+    Modify Section 7.1.5 Fragment Shader Special Variables to Chapter 7 Built-in Variables (p. 145)
+
+    (add the followings after 'gl_HelperInvocation')
+
+    in uvec2 gl_TileOffsetQCOM
+    in uvec3 gl_TileDimensionQCOM
+    in uvec2 gl_TileApronSizeQCOM
+
+    (add to the end of Section 7.1.5. Fragment Shader Special Variables)
+
+    The input variable gl_TileOffsetQCOM is filled with the framebuffer
+    coordinates of the top-left texel of the current tile.
+
+    The input variable gl_TileDimensionQCOM represents the tile dimension in pixels (x and y)
+    as well as the number of layers (z).  The gl_TileDimensionQCOM will report the same tile
+    dimensions for all tiles in a rendering pass, including "edge tiles" that may only be
+    partially rendered because the tile boundaries extend beyond the edges of the framebuffer
+    or the Vulkan API "renderArea".
+
+    The input variable gl_TileApronSizeQCOM represents the widths of an apron (in X and Y directions).
+
+    
+    Modify Section 7.1.6 Compute Shader Special Variables to Chapter 7, Built-in Variables (p. 148)
+
+    (add the followings after 'gl_LocalInvocationIndex')
+
+    in uvec2 gl_TileOffsetQCOM
+    in uvec3 gl_TileDimensionQCOM
+    in uvec2 gl_TileApronSizeQCOM
+
+    (add to the end of Section 7.1.6. Compute Shader Special Variables)
+
+    The input variable gl_TileOffsetQCOM is filled with the framebuffer
+    coordinates of the top-left texel of the current tile.
+
+    The input variable gl_TileDimensionQCOM represents the tile dimension in pixels (x and y)
+    as well as the number of layers (z).  The gl_TileDimensionQCOM will report the same tile
+    dimensions for all tiles in a rendering pass, including "edge tiles" that may only be
+    partially rendered because the tile boundaries extend beyond the edges of the framebuffer
+    or the Vulkan API "renderArea".
+
+    The input variable gl_TileApronSizeQCOM represents the widths of an apron (in X and Y directions). 
```

Comment:
I think it would be appropriate to replace this text with something that just says that they;re defined the same as in fragment shaders?

Reply:
Done.